openedx/adg/common/course_meta/helpers.py

```diff
@@ -0,0 +1,24 @@
+"""
+All the helper methods for course meta app
+"""
+from contextlib import suppress
+
+from django.apps import apps
+from django.db.utils import OperationalError
+
+
+def next_course_short_id():
+    """
+    Create next unique course short id. Course short id's start with three digits i.e. 100
+    Returns:
+        Course short if, starting from 100
+    """
+    course_short_id = 100  # Start with (smallest) three digit id
+    CourseMeta = apps.get_model('course_meta', 'CourseMeta')  # noqa (N807), camelcase name is good for searchability
+
+    with suppress(OperationalError):
```

Comment:
It occurs while applying migration. During migration, default is triggered before creating of `course_short_id` field/column. Default function make query on the same model, and model throws error because `course_short_id` is not yet a column in table